pkg/operator/options/options.go

```diff
@@ -44,6 +49,27 @@ type Options struct {
 	EnableLeaderElection bool
 	MemoryLimit          int64
 	LogLevel             string
+	BatchMaxDuration     time.Duration
+	BatchIdleDuration    time.Duration
+	DriftEnabled         bool
```

Comment:
Does it make sense at all to have a `Gates` field here that stores all the feature gates and to pre-populate these gates with defaults at the time that we initialize the `NewOptions()`?

Reply:
Sure, I think that would be a good improvement 👍